rbx_binary/src/deserializer/state.rs

```diff
@@ -1439,6 +1453,57 @@ rbx-dom may require changes to fully support this property. Please open an issue
                     });
                 }
             },
+            Type::Content => match canonical_type {
+                VariantType::Content => {
+                    let mut source_types = vec![0; type_info.referents.len()];
+                    chunk.read_interleaved_i32_array(&mut source_types)?;
+
+                    let uri_count = chunk.read_le_u32()? as usize;
+                    let mut uris = VecDeque::with_capacity(uri_count);
+                    for _ in 0..uri_count {
+                        uris.push_front(chunk.read_string()?);
+                    }
+
+                    let object_count = chunk.read_le_u32()? as usize;
+                    let mut objects: VecDeque<i32> = vec![0; object_count].into();
+                    chunk.read_referent_array(objects.make_contiguous())?;
+
+                    let external_count = chunk.read_le_u32().unwrap() as usize;
+                    // We are advised by Roblox to just ignore this, as it's
+                    // meant for internal use. If we want to use it in the
+                    // future, it's a referent array.
```

Comment:
If you want to say it's sometimes used for patching up references to the original object if it still exists, that's cool. If you ever see referents >= `objects.len()` in general that's usually what they're for, patching up external refs.